src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -43,7 +43,7 @@ interface MapProps {
 
 export const Map: React.FC<MapProps> = ({
   user,
-  profileData,
+  profileData = {},
```

Comment:
Since the rest of the code that uses this already appears to be written to handle it being potentially `undefined`, we could probably avoid defaulting it here.

Alternately, we could change the other usages to always expect it to be defined.

Reply:
I think my preference would be to leave it as being potentially `undefined`, and remove the default here (as then we are able to easily check whether they have existing data or not); but i'm open to counter arguments why that wouldn't be the best way.